apps/web/vibes/soul/sections/product-detail/product-detail-form.tsx

```diff
@@ -53,6 +54,7 @@ export function ProductDetailForm<F extends Field>({
   quantityLabel = 'Quantity',
   incrementLabel = 'Increase quantity',
   decrementLabel = 'Decrease quantity',
+  disabled = false,
```

Comment:
Do we want to use this just for the CTA and name it `ctaDisabled`?

Also, we probably want some form of consistency between the prop names in this component and the section component.

Reply:
I wonder if we should use the naming `addToCart` since that's really what this is designed for. Or are there other instances where perhaps you could immediately check out from a PDP? 🤔